docs/developer/running-saleor/s3.mdx

```diff
@@ -11,13 +11,14 @@ This integration allows you to delegate storing such files to [Amazon’s S3 ser
 | Variable Name              | Description                                     |
 |----------------------------|-------------------------------------------------|
 | `AWS_ACCESS_KEY_ID`        | Your AWS access key.                            |
-| `AWS_SECRET_ACCESS_KEY`    | Your AWS secret access key.                     |
+| `AWS_DEFAULT_ACL`          | Default [Access Control List](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html) to uploaded files. |
```

Comment:
When reading this I had a concern: "Default to what? Is Saleor trying to do something I did not ask it to: setting the ACL?" - can you mention (if my guess is right) that if none is provided, it will be leveraged to the s3 settings? (meaning that Saleor does not try to guess anything, pushes the file without setting any ACL)

Reply:
Good feedback, thanks!